mlir/test/Target/LLVMIR/llvmir.mlir

```diff
@@ -2396,3 +2396,9 @@ llvm.func @zeroinit_complex_local_aggregate() {
 llvm.linker_options ["/DEFAULTLIB:", "libcmt"]
 //CHECK: ![[MD1]] = !{!"/DEFAULTLIB:", !"libcmtd"}
 llvm.linker_options ["/DEFAULTLIB:", "libcmtd"]
+
+// -----
+
+// Translation is currently very slow so the test is not enabled.
+//llvm.mlir.global common @big_(dense<0> : vector<4294967296xi8>) {addr_space = 0 : i32} : !llvm.array<4294967296 x i8>
```

Comment:
This is taking a very long time to translate so I'm wondering if we should use another form of attribute or if the translation is the problem. Any insight?

Reply:
So an array of 4G constant (16GB  memory just for the array of pointers) is gonna kill a lot of machines out there :)

I don't have a suggestion here though right now. I think you can already that the type change with a simple round-trip, without testing the translation?

Reply:
Yeah unfortunately it is smth in fortran :(  I guess round trip will be enough.